src/renderer/components/Modal/ModalPeer.vue

```diff
@@ -133,7 +133,9 @@ export default {
   mounted () {
     if (this.currentPeer) {
       if (this.currentPeer.host) {
-        this.form.host = this.currentPeer.host
+        const scheme = this.currentPeer.isHttps ? 'https://' : 'http://'
```

Comment:
You can move this line to the top, because `line 140` has the same code.

Example:

```
  mounted () {
    if (this.currentPeer) {
      const scheme = this.currentPeer.isHttps ? 'https://' : 'http://'
      
      if (this.currentPeer.host) {
        const hostname = new URL(this.currentPeer.host).hostname
        this.form.host = scheme + hostname
      } else if (this.currentPeer.ip) {
        this.form.host = `${scheme}${this.currentPeer.ip}`
      }

      if (this.currentPeer.port) {
        this.form.port = this.currentPeer.port
      }
    }
  }
```

Reply:
Done!